modin/pandas/indexing.py

```diff
@@ -273,15 +284,14 @@ def _write_items(self, row_lookup, col_lookup, item):
                 result_oid = writer.remote(block_oid, row_idx, col_idx,
                                            item_to_write)
 
-                if self.is_view:
-                    self.df._block_partitions_data[row_blk,
-                                                   col_blk] = result_oid
-                else:
-                    self.df._block_partitions[row_blk, col_blk] = result_oid
+                self.df._block_partitions[row_blk, col_blk] = result_oid
 
                 col_item_index += col_len
             row_item_index += row_len
 
+        if self.is_view:
+            warn(_SETTING_WITHOUT_COPYING_WARING)
```

Comment:
Can this warning be imported from pandas?

Reply:
Sadly, no... Pandas uses inline warning text:
https://github.com/pandas-dev/pandas/blob/27ebb3e1e40513ad5f8919a5bbc7298e2e070a39/pandas/core/generic.py#L2778-L2805

Reply:
Ahh weird